### ISSUE TITLE
MD / Genesis Games via Mega Everdrive PRO (by Krikzz) are crashing

### ISSUE THREAD

**Issue by BambooShadow** [TRP-Retromods/Megadrive-Genesis-DFO-Switchless#1]
I have a MegaDrive 2 ("MD2 VA0 PAL" mainboard) console with the switchless DFO-mod. It works perfectly well with original cartridges and also with my Mega Everdrive X5.

However, when I use it with the Mega Everdrive PRO, Genesis and Mega Drive Games won't work. They always crash after about 30 seconds of gameplay.
SMS Roms and also Mega CD Games work perfectly fine on that console.

The Mega Everdrive PRO works perfectly well with MD/Genesis Games on an unmodded console.

It is really only the MD/Genesis Roms on the Switchless Console, that are crashing.

Any idea why this happens?

**Comment by BambooShadow** [comment]
Here is some gameplay, demonstrating the problems:
1.) game randomly resets (after about 30-60seconds): https://clips.twitch.tv/NeighborlyBoldCaribouFUNgineer-_7I8X7ZQi7jrnDK6
2.) game gets stuck (always seems to be at similar places, but also around 60 seconds after booting the game): https://clips.twitch.tv/LovelyDeadPizzaPMSTwin-uSxKO6tNS-BqzJaD
3.) game randomly crashes (happens sometimes, sometimes not...): https://clips.twitch.tv/DifficultOptimisticPonyRalpherZ-DoRca4VOHG2o83Cp

Some games seem to run fine, without crashing. But most games, show some problems sooner or later. Region doesn't matter, happens to PAL but also NTSC-U & NTSC-J roms.

**Comment by BambooShadow** [comment]
@TRP-Retromods this can be closed. After a very extensive analysis, the final solution was absolute prepostorous: re-install the console with the bottom metal/RF-shield included and everything works perfectly fine again! Shouldn't have listened to the 3BP-guide, telling me I could leave out the bottom shield :'-(.

**Comment by rfmerrill** [comment]
> [@TRP-Retromods](https://github.com/TRP-Retromods) this can be closed. After a very extensive analysis, the final solution was absolute prepostorous: re-install the console with the bottom metal/RF-shield included and everything works perfectly fine again! Shouldn't have listened to the 3BP-guide, telling me I could leave out the bottom shield :'-(.

I've never heard of this happening. In nearly every scenario the RF shield makes absolutely no difference in functionality.

However, I have observed that the Mega EverDrive Pro drives the data bus marginally, and on a minority of consoles I had glitchiness and crashes that were solved by removing the capacitor bank CA1 from the data lines (on the console, not the ED), which increased the rise time just enough to make it stable again. It's possible if you have a situation where it's just on the edge like this, the shield could make the difference.
